packages/desktop/components/DesktopLayout/index.tsx

```diff
@@ -100,7 +100,7 @@ export const DesktopLayout = !window.wowarenalogs
           >
             <AuthProvider>
               <LocalCombatsContextProvider>
-                <div className="w-screen h-screen flex flex-col bg-base-300">
+                <div className="w-screen h-screen flex flex-col bg-base-300 overflow-hidden">
```

Comment:
Not sure this resolved it entirely for me -- still seeing horizontal scroll on the Summary page

![image](https://user-images.githubusercontent.com/15525519/204152452-9360da0c-29b9-4c37-8128-9b442389b4d8.png)